### ISSUE TITLE
language support

### ISSUE THREAD

**Issue by hkkongou** [Minebench/EntityDetection#4]
#### Is your feature request related to a problem? Please describe.

This is no a problem but instead, is a suggestion.
I would like to use ``/detect list `` command to replace clearlagg ``/lagg chunk`` command, i think entitydetection detect much better, however, i found out that there is no file for me to change the language displayed in the GUI.

#### Describe the solution you'd like

Create a folder with config.yml or a lang folder that contain default language file, people can change it manually.

#### Describe alternatives you've considered

it comes with auto language detection, it display the language that user is using

#### Additional context

![image](https://user-images.githubusercontent.com/61608872/111326714-c05e5480-86a7-11eb-9722-2cadf35608b0.png)

as i don't open the permission for players to tp, the list of top number mobs in certain chunk may not be that useful.

I also come up with another solution, player can set the format displayed in the``/detect list``, <RANK> as the entry number placeholder and something like that, the display can be configure by users and make:

``<RANK> | Chunk: <x> , <z> | World: <worldname> | total: <entities>`` 

so that it looks like
``① | Chunk: 0 , 2 | World: world | total: 7`` 

The current setting is like this 🤔
``<RANK>: <x>, <z>: <worldname> <entities> <detail>``

Maybe i think too much, but if the GUI is also nice, this can total win clearlagg in terms of entity number display and light weight.

**Comment by Phoenix616** [comment]
Yeah, that has been on my todo list for a while, will be done with [lang](https://github.com/Phoenix616/lang) and [MineDown](https://github.com/Phoenix616/MineDown) eventually.

**Comment by SlimeDog** [comment]
This would also be nice to allow me to change the format/colors.

**Comment by yepikaiye** [comment]
+1, just recently discovered it, one of the best plugins when it comes to "antilag", would love to see language support and more love for this plugin.

might even tip if it gets added 🙏
